### PR TITLE
Unit tests for Magento\Csp\Model\Mode\ConfigManager and Magento\Csp\Observer\Render

### PR DESCRIPTION
**Pull request by karyna-t**
### Description (*)
Covers \Magento\Csp\Model\Mode\ConfigManager and \Magento\Csp\Observer\Render with a unit tests.

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [x] All new or changed code is covered with unit/integration tests (if applicable)
 - [x] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Review by lbajsarowicz — APPROVED** [review]
Another good contribution, actually.

File: app/code/Magento/Csp/Test/Unit/Observer/RenderTest.php [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+        /** @var MockObject|Observer $eventObserverMock */
+        $eventObserverMock = $this->createMock(Observer::class);
+        $eventObserverMock->expects($this->once())->method('getEvent')->willReturn($eventMock);
+
+        $cspRendererMock = $this->createMock(CspRendererInterface::class);
+        $cspRendererMock->expects($this->once())->method('render');
+
+        $objectManagerHelper = new ObjectManager($this);
+        /** @var MockObject|Render $renderObserver */
```
Comment (eduard13) [inline app/code/Magento/Csp/Test/Unit/Observer/RenderTest.php]: Hi @karyna-tsymbal-atwix, just one notice here. Could you please remove the MockObject from here, as this is the real class that you test.
Thank you.

Comment (lbajsarowicz) [reply app/code/Magento/Csp/Test/Unit/Observer/RenderTest.php]: Nice catch. Thanks @eduard13 you verified that :-)

Comment (karyna-t) [reply app/code/Magento/Csp/Test/Unit/Observer/RenderTest.php]: @eduard13  thanks for the note! fixed.

**Review by lbajsarowicz — APPROVED** [review]

**Comment by engcom-Bravo** [comment]
:heavy_check_mark: QA Passed

The added tests pass
confmanag

render

**Review by eduard13 — CHANGES_REQUESTED** [review]
Hi @karyna-tsymbal-atwix, could you please fix the merging conflicts? Looks like the `Magento\Csp\Model\Mode\ConfigManager` class has been already covered by Unit Test.

**Review by eduard13 — APPROVED** [review]

**Pull request merged**